src/omero_cli_render.py

```diff
@@ -396,6 +397,11 @@ def _configure(self, parser):
                 help="Local file or OriginalFile:ID which specifies the "
                      "rendering settings")
 
+        set_cmd.add_argument(
+            "--batch", type=int, default=1,
```

Comment:
Have you considered a higher default?

Reply:
Could do. Just didn't want to change the default behaviour if the new option is omitted.